File: Módulo_I/Ex04_PY/disciplina.py
class Disciplina:

    def __init__(self, codigo, nome, valor):
        
        self.codigo = codigo
        self.nome = nome
        self.valor = valor

    def __str__(self):
        
        return f'Código: {self.codigo} Nome: {self.nome} Valor: {self.valor}'

File: Modulo_I/faculdade/JAVA/Disciplina.java
class Disciplina {

    private int codigo;
    private String nome;
    private double valor;

    public Disciplina(){}

    public Disciplina(int codigo, String nome, double valor){

        this.codigo = codigo;
        this.nome = nome;
        this.valor = valor;
    }

    public int getCodigo() { return codigo; }
    public String getNome() { return nome; }
    public double getValor() { return valor; }

    public void setCodigo(int codigo) { this.codigo = codigo; }
    public void setNome(String nome) { this.nome = nome; }
    public void setValor(double valor) { this.valor = valor; }

    @Override
    public String toString(){

        return "Código: " + this.codigo + " | Nome: " + this.nome + " | Valor: " + this.valor;
    }
}